tests/karma/unit/crmCaseTypeSpec.js

```diff
@@ -15,22 +23,51 @@ describe('crmCaseType', function() {
     inject(function(crmJsonComparator) {
       crmJsonComparator.register(jasmine);
     });
+
+    caseTypeCategories = {
+      values: [
+        {
+          "id": "1170",
+          "option_group_id": "153",
+          "label": "Workflow",
+          "value": "1",
+          "name": "WORKFLOW",
+          "filter": "0",
+          "is_default": "0",
+          "weight": "1",
+          "is_optgroup": "0",
+          "is_reserved": "1",
+          "is_active": "1"
+        },
+        {
+          "id": "1171",
+          "option_group_id": "153",
+          "label": "Vacancy",
+          "value": "2",
+          "name": "VACANCY",
+          "filter": "0",
+          "is_default": "0",
+          "weight": "2",
+          "is_optgroup": "0",
+          "is_reserved": "1",
+          "is_active": "1"
+        }
+      ]
+    };
   });
 
+  beforeEach(inject(function(_$controller_, _$httpBackend_, $compile, $rootScope, $timeout) {
```

Comment:
In core, this code was inside the `CaseTypeCtrl`. Why change this? Not sure if core will accept.

Reply:
Ah, sorry, I should have explained this. They didn't have any tests for `CaseTypeListCtrl`, since all controllers are shared between a single `crmCaseType.js` file, I decided to also add the tests for `CaseTypeListCtrl` on this file.

I moved this code so both tests could share these variables, otherwise I'd have to repeat them in the list test.

I'll make sure to explain this to core, I'm sure they'll be fine with it since at least we are covering a few more tests that they were missing.